Header header
float64 travelled_distance